packages/page-accounts/src/Accounts/Account.tsx

```diff
@@ -86,8 +87,9 @@ const transformRecovery = {
   transform: (opt: Option<RecoveryConfig>) => opt.unwrapOr(null)
 };
 
-function Account ({ account: { address, meta }, className = '', delegation, filter, isFavorite, proxy, setBalance, toggleFavorite }: Props): React.ReactElement<Props> | null {
+function Account ({ account: { address, meta }, className = '', delegation, filter, isEven, isFavorite, proxy, setBalance, toggleFavorite }: Props): React.ReactElement<Props> | null {
   const { t } = useTranslation();
+  const [isExpanded, setIsExpanded] = useState(false);
```

Comment:
```suggestion
  const [isExpanded, toggleIsExpanded] = useToggle();
```
Then you can get rid of `handleOnExpandIconClick` and call `toggleIsExpanded` directly, as it's memoized in the hook. I believe similar change can be made in `Bounty.tsx`.

Reply:
✅